constraint_utils/constraint_basics.py

```diff
@@ -425,7 +460,46 @@ def get_proportion_observed(exome_ht: hl.Table, context_ht: hl.Table, mutation_h
                             recompute_possible: bool = False, remove_from_denominator: bool = True,
                             custom_model: str = None, dataset: str = 'gnomad',
                             impose_high_af_cutoff_upfront: bool = True, half_cutoff = False) -> hl.Table:
-
+    """
+    Compute the expected number of variants using plateau models and coverage model.
+
+    This function sums the number of possible variants times the mutation rate for all variants, and apply the calibration
+    model separately for CpG transitions and other sites. For sites with coverage lower than coverage cutoff, the value got 
+    from previous step is multiplied by the coverage correction factor. These values are summed across the set of variants 
+    of interest to obtain the expected number of variants.
```

Comment:
I felt this description is too wordy, so I added a brief version below. However, the brief one is not that formal, so I wonder which one is better.